main.go

```diff
@@ -425,6 +425,8 @@ func main() {
 	migrate.TryMigrateToV12()
 	// 尝试修正log_items表的message字段类型
 	_ = migrate.LogItemFixDatatype()
+	// v131迁移历史设置项到自定义文案
+	_ = migrate.V131DeprecatedConfig2CustomText()
```

Comment:
可以考虑接下error之后打个日志. 上面的`LogItemFixDatatype`也不妨一样处理

```suggestion
	if errMig := migrate.V131DeprecatedConfig2CustomText(); errMig != nil {
    logger.Error("xxx迁移过程中出现错误", errMig)
  }
```